icu4c/source/common/cjmlbe.h

```diff
@@ -0,0 +1,108 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef CJMLBREAKENGINE_H
+#define CJMLBREAKENGINE_H
+
+#include "unicode/uniset.h"
+#include "unicode/utext.h"
+#include "hash.h"
+#include "uvectr32.h"
+
+
+U_NAMESPACE_BEGIN
+
+#if !UCONFIG_NO_NORMALIZATION
```

Comment:
I think this should be
#if !UCONFIG_NO_BREAK_ITERATION
instead of UCONFIG_NO_NORMALIZATION

Reply:
Done